hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ResourceAttributeEnricher.java

```diff
@@ -77,4 +86,15 @@ private boolean isValidEvent(Event event) {
         && (event.getAttributes() != null)
         && (event.getAttributes().getAttributeMap() != null);
   }
+
+  private String getDeploymentType(String hostName) {
+    for (Deployment d : Deployment.values()) {
+      if (d == Deployment.UNRECOGNIZED) {
```

Comment:
whats this condition handling for ? ( d==uncrecognized ) ???

Reply:
.values() return unrecognized as enum value once iteration is done over all valid values. 
https://developers.google.com/protocol-buffers/docs/reference/java-generated